src/sidebar/store/modules/selection.js

```diff
@@ -502,6 +508,10 @@ function focusModeUserPrettyName(state) {
   }
 }
 
+function getSelectedAnnotationMap(state) {
+  return state.selection.selectedAnnotationMap;
```

Comment:
The selectors don't have to return the same type of value as the internal state. It might be more convenient to return an array of selected annotations here and call this `selectedAnnotations`.

Reply:
Oh, yes, I want to reassess how this is named, and a couple of other state properties within. I just wanted to hold off for another round of changes.